Tests/LibWeb/Text/expected/css/getComputedStyle-print-all.txt

```diff
@@ -104,6 +104,7 @@ margin-left: 8px
 margin-right: 8px
 margin-top: 8px
 mask: none
+mask-type: luminance
```

Comment:
Commit message: It's `mask-type` not `mask-kind`.

Reply:
Fixed, thanks!